e2e-test/trace/src/integrationTest/resources/raw-span-e2e-pipeline-latest-release.yml

```diff
@@ -2,7 +2,7 @@ entry-pipeline:
   source:
     otel_trace_source:
       ssl: false
-  prepper:
+  processor:
```

Comment:
I think this is causing the e2e to fail. b/c this config is used for latest release(1.1) of data-prepper.